roxctl/utils/help_formatter.go

```diff
@@ -0,0 +1,139 @@
+package utils
+
+import (
+	"io"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// FormatHelp formats command help.
+// FTR, kubectl uses a similar, but even more sophisticated custom formatter:
+// https://github.com/kubernetes/kubectl/blob/master/pkg/util/templates/templater.go
+func FormatHelp(c *cobra.Command, _ []string) {
+	w := makeHelpWriter(
+		makeFormattingWriter(&cobraWriter{c}, 80))
```

Comment:
We already have a term size https://github.com/stackrox/stackrox/blob/93d69312822ca7e1775a6172bfb6a18fd54677ea/roxctl/central/db/restore/v2_restorer.go#L163
How about using it